### DISCUSSION TITLE
安装过程中出错

### DISCUSSION THREAD

**Discussion by Janghhy** [STVIR/pysot#355] [General]
在运行python setup.py build_ext --inplace这句命令时
出现了这个错误error: Error executing cmd /u /c "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x86_amd64 && set 
网上也找不到解决方案 方便解答一下吗

**Comment by ZhiyuanChen** [comment]
这看上去像是一个visual studio的问题。
参见 https://github.com/microsoft/vcpkg/issues/5587
